packages/site/src/layouts/base.astro

```diff
@@ -20,9 +20,14 @@ const navData = searchData.map(o => pick(o, ["name", "subCategories"]))
 <html class={themes[`${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}Theme`]} lang="en">
   <head>
 		{fontHeadTags}
-    <title>{`${title} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`}</title>
+    	<title>{`${title} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`}</title>
```

Comment:
Nitpick: the indentation in this file seems a bit strange 🙂

Reply:
I believe it doesn't respect linting at this point, on vscode, it is formatted nicely but on GitHub, the indentation for the title shoots off.